core/templates/core/index.html

```diff
@@ -87,6 +87,15 @@ <h4 class="modal-title" id="announcementModalLabel">Add Announcement</h4>
             </div>
             {% endif %}
         </div>
+        <div class="col-lg-4">
+            <h2>Upcoming Events</h2>
+            <ul class="list-group">
+            {% for event in events %}
+                <li class="list-group-item list-group-item-action" onclick="window.location='{{ event.get_url }}';">
+                    {{ event.date }}, {{ event.time }}:  {{ event.name }}
```

Comment:
I think that this should be in the same format as the social page: Name, date, time. Also, the text should be blue like the social page imo